src/timeAgo.test.ts

```diff
@@ -220,4 +220,9 @@ describe('timeAgo', () => {
 		expect(timeAgo(eightDaysAgo)).toBe('9 Nov 2019');
 		expect(timeAgo(aWhileBack)).toBe('2 Apr 2017');
 	});
+
+	it('returns false on future dates', () => {
+		const tomorrow = new Date(Date.UTC(2019, 10, 18, 10, 0, 0)).getTime();
+		expect(timeAgo(tomorrow)).toBe(false);
+	});
```

Comment:
👍 nice

Reply:
`false` 😆

Reply:
should it return `false` or should it error?

Reply:
> Or perhaps it could return 'now' for future dates? I think a date in the future is at least theoretically possible, it's hard to keep server time in sync, I've definitely seen thing like "Last updated in 2 minutes" on other platforms. So maybe we want to return the most useful response for the most likely scenario, where the date is a few seconds in the future? -@oliverlloyd 
>
> Or return “recently” and warn inside the console? It’s a soft fallback that isn’t wrong for the user, while remaining distinctly different to other values. -@mxdvl
>
> I like how this provides a graceful failover in all cases 👍 . I guess the question is, do we want to trigger errors on future dates so that we discover them or handle them silently?- @oliverlloyd 

I’m thinking we should handle silently for now? I don’t think an error in a timezones should stop JS from executing.

Reply:
it may be beyond the scope of this PR, but perhaps this module should be called `timeFromNow` or `toRelativeTime` and handle past and future dates?

Reply:
I think the final solution to this can be addressed in a subsequent PR :D